brage-import/src/test/java/cucumber/features/transformers/PublishedFileTransformer.java

```diff
@@ -46,7 +47,7 @@ public static PublishedFile toPublishedFile(Map<String, String> entry) {
             parseRightsRetentionStrategy(entry),
             entry.getOrDefault("legalNote", null),
             parseDate(entry.get("publishedDate")),
-            null
+            new UploadDetails(null, null)
```

Comment:
Did not merge main before I bumped datamodel, so had to resolve all this merge conflicts, but it does absolutely nothing.